packages/batch/src/types.ts

```diff
@@ -248,13 +248,11 @@ type ParsedRecord<TRecord, TPayload, TOldPayload = TPayload> = TRecord extends {
  * Parts of the parser integration within BatchProcessor rely on Zod for schema transformations,
  * however some other parts also support other Standard Schema-compatible libraries.
  *
- * To avoid forcing a direct dependency on Zod, we use `any` here, which is not ideal but necessary.
+ * To avoid forcing a direct dependency on Zod, we use `unknown` here, which is not ideal but necessary.
  *
  * The vendor is checked at runtime to ensure Zod is being used when required using `StandardSchemaV1['~standard'].vendor`.
- *
- * biome-ignore lint/suspicious/noExplicitAny: using `any` to avoid direct dependency on 'zod'
  */
-type RuntimeZodType = any;
+type RuntimeZodType = unknown;
```

Comment:
I believe we now use this alias in a single place - I think we can remove it and keep the description in the `createExtendedSchema` function description, so Sonar is also happy.